docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    aggregated = run_db.list_alert_activations().group_by("severity")
</pre><p>it's confusing that you are using <code>aggregated</code> as the result of <code>group_by</code>, and the "Aggregate By" section is only later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>